scripts/vippy/rest.py

```diff
@@ -329,8 +331,10 @@ def update_rest_files(override_type_name=None):
 
         prefix = "multi-{}".format(mode)
         for type_name in type_names:
-            _log.debug("updating rest files for type: {0}".format(type_name))
             data_type = common.get_type(type_map, type_name)
+            if mode == data_type.skip_element_on:
```

Comment:
And the same here, skipping when creating the individual (as opposed to the single-page) rst file.